torch/nn/modules/module.py

```diff
@@ -417,6 +417,12 @@ def forward(self, x):
         As per the example above, an ``__init__()`` call to the parent class
         must be made before assignment on the child.
 
+    .. note::
+        Avoid performing differentiable operations in the ``__init__`` method of a Module.
+        Otherwise, moving tensors to different devices (e.g., using ``tensor.to('cuda')``)
+        may not be properly tracked by the autograd system, causing ``tensor.grad`` to remain
+        on the original device.
```

Comment:
I think it might make sense to just say 
```suggestion
        Avoid performing differentiable operations in the ``__init__`` method of a Module.
```

As I'm not sure the issue with moving tensors to different devices is the *only* interaction with autograd that might happen

cc @albanD does this sound reasonable or is there a better articulation here?

Reply:
The shorter message sounds good!